os/arch/arm/src/armv7-m/up_assert.c

```diff
@@ -268,34 +268,37 @@ static void up_dumpstate(void)
 	istackbase = (uint32_t)&g_intstackbase;
 	istacksize = (CONFIG_ARCH_INTERRUPTSTACK & ~3);
 
-	/* Show interrupt stack info */
-
 	lldbg("sp:     %08x\n", sp);
-	lldbg("IRQ stack:\n");
-	lldbg("  base: %08x\n", istackbase);
-	lldbg("  size: %08x\n", istacksize);
-#ifdef CONFIG_STACK_COLORATION
-	lldbg("  used: %08x\n", up_check_intstack());
-#endif
-
-	/* Does the current stack pointer lie within the interrupt
-	 * stack?
-	 */
-
-	if (sp <= istackbase && sp > istackbase - istacksize) {
-		/* Yes.. dump the interrupt stack */
-
-		up_stackdump(sp, istackbase);
-	}
 
-	/* Extract the user stack pointer if we are in an interrupt handler.
-	 * If we are not in an interrupt handler.  Then sp is the user stack
-	 * pointer (and the above range check should have failed).
+	/* Show the interrupt number, handler and stack information
+	 * if we are in an interrupt handler
 	 */
 
 	if (current_regs) {
-		sp = current_regs[REG_R13];
-		lldbg("sp:     %08x\n", sp);
```

Comment:
Those two lines are removed. No need anymore?

Reply:
I checked, I think this will be required , I will modify